.github/workflows/test_rust.yml

```diff
@@ -1,5 +1,9 @@
 name: Test Rust
 
+
+env:
+  ASSERT_PLAYERGLOBAL: 1
```

Comment:
I think it's reasonable enough to require developers have Java installed. It will allow us to not checkout `library.swf` into git, and remove this CI step. Also, can you please add the Java requirement in `README.md` under the "Building from source" section?